src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/emails/administration/AdministrationEmailServiceTest.kt

```diff
@@ -162,12 +162,12 @@ class AdministrationEmailServiceTest {
 
   @Test
   fun `should send no emails when no new rooms`() {
-    whenever(prisonsService.getListOfPrisons(true)) doReturn emptyList()
```

Comment:
The test fixture set up was not quite right therefore the expected assertions were also not quite right.